app/src/main/res/layout/view_fade_omnibar.xml

```diff
@@ -152,6 +141,17 @@
                                     android:scaleType="center"
                                     android:src="@drawable/ic_find_search_20_a05" />
 
+                                <ImageView
+                                    android:id="@+id/backIcon"
+                                    android:layout_width="@dimen/toolbarIcon"
+                                    android:layout_height="@dimen/toolbarIcon"
+                                    android:layout_gravity="center"
+                                    android:contentDescription="@string/back"
+                                    android:gravity="center"
+                                    android:scaleType="center"
+                                    android:src="@drawable/ic_arrow_left_24e_omnibar"
```

Comment:
touch feedbacks are free, don’t be that guy and splash on some please

`android:background="?selectableItemBackgroundBorderless"`